src/sync/order_rewards.py

```diff
@@ -56,23 +56,70 @@ def write_sync_data(self) -> None:
 def sync_order_rewards(
     aws: AWSClient, fetcher: OrderbookFetcher, config: SyncConfig, dry_run: bool
 ) -> None:
-    """App Data Sync Logic"""
+    """Order Rewards Data Sync Logic"""
+    sync_table = SyncTable.ORDER_REWARDS
+    block_range = BlockRange(
+        block_from=last_sync_block(
+            aws,
+            table=sync_table,
+            genesis_block=15719994,  # First Recorded Order Reward block
+        ),
+        block_to=fetcher.get_latest_block(),
+    )
+    sync_orderbook_data(
+        aws,
+        block_range,
+        config,
+        dry_run,
+        sync_table=sync_table,
+        data_list=OrderRewards.from_pdf_to_dune_records(
+            fetcher.get_order_rewards(block_range)
+        ),
+    )
+
 
+def sync_batch_rewards(
+    aws: AWSClient, fetcher: OrderbookFetcher, config: SyncConfig, dry_run: bool
+) -> None:
+    """Batch Reward Sync Logic"""
+    sync_table = SyncTable.BATCH_REWARDS
     block_range = BlockRange(
         block_from=last_sync_block(
             aws,
-            table=SYNC_TABLE,
+            table=sync_table,
             genesis_block=15719994,  # First Recorded Order Reward block
```

Comment:
Shouldn't this be a different block?

Reply:
Yes absolutely! Great catch (copy-pasta).

Reply:
We may not actually know this block till later but its really just any lower bound so I will choose a recent block from today for now and leave a TODO here.